src/blocks/icon-list/style.scss

```diff
@@ -102,4 +102,11 @@
 		grid-template-rows: auto;
 		display: grid;
 	}
-}
\ No newline at end of file
+}
+
+.kt-svg-icon-list-level-0 { padding-left: 0rem; }
```

Comment:
Are the indentions okay this way ?

Reply:
Yeah this looks fine to me! I added them to the `editor.scss` so they would be visible in the editor too